src/Menu/Client.java

```diff
@@ -0,0 +1,47 @@
+package Menu;
+
+import java.util.Date;
+
+public class Client {
+
+    private String name = "";
+    private Date birthDate = new Date();
+    private String userCPF;
+    private String email =  "";
+    private Date registerDate = new Date();
+    Address address = new Address();
+
+
+    public Client() {
+    }
+
+    //client construcotor
+    public Client(String name, Date birthDate, String userCPF, String email, Date registerDate, Address addres){
```

Comment:
```suggestion
    public Client(String name, Date birthDate, String userCPF, String email, Address address){
```
Esse register date vc nao cadastra aqui, quando criar o objeto vc ja define.